src/internals/ExtendedJSX.ts

```diff
@@ -22,9 +22,11 @@ export function getJsxInfo(source: any): JsxInfo {
   return {source};
 }
 
+export type Severity = 'warn' | 'error';
```

Comment:
I'd expect to have an option like `'none'`

Reply:
Future